Lesson04/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Cabin&display=swap" rel="stylesheet">
+    <title>Lesson 4: Assignment 1</title>
+    <style>
```

Comment:
I know it's not the point of this excercise, but no inline styles. :)

Reply:
ah, what's a better alternative to referencing a web font?